CMakeLists.txt

```diff
@@ -90,11 +90,18 @@ endif()
 
 set(H3_COMPILE_FLAGS "")
 set(H3_LINK_FLAGS "")
+option(ENABLE_WARNINGS "Enables compiler warnings" ON)
+if(ENABLE_WARNINGS)
+    if(WIN32)
```

Comment:
This is wrong, `/W2` is a MSVC flag and you should check [`MSVC`](https://cmake.org/cmake/help/latest/variable/MSVC.html) instead.  MSVC isn't the only Windows compiler available.

Reply:
Oh, sorry! I've opened an issue for this: #980

I'll open a PR too.

Reply:
#981